x/reports/types/models/report.go

```diff
@@ -44,7 +44,7 @@ type Reports []Report
 
 // String implements stringer
 func (reports Reports) String() string {
-	out := "Type - Message - Sender\n"
+	out := "Type - Message - ReceivingUser\n"
```

Comment:
I think it's better to leave it as it was

```suggestion
	out := "Type - Message - Sender\n"
```